x-pack/plugins/security/server/plugin.ts

```diff
@@ -93,7 +104,8 @@ export interface PluginStartDependencies {
 export class Plugin {
   private readonly logger: Logger;
   private securityLicenseService?: SecurityLicenseService;
-  private authc?: Authentication;
+  private authenticationStart?: AuthenticationServiceStart;
+  private authz?: AuthorizationServiceSetup;
```

Comment:
optional nit: I'd prefer to either abbreviate both, or expand both for consistency:

1) `authenticationStart` & `authorizationSetup`
2) `authc` & `authz`
3) `authcStart` & `authzSetup`

Reply:
Yep, makes sense, leaning towards the first option.